src/main/java/org/openhab/binding/zwave/handler/ZWaveThingHandler.java

```diff
@@ -264,7 +268,7 @@ void initialiseNode() {
                     try {
                         dataType = DataType.valueOf(bindingType[2]);
                     } catch (IllegalArgumentException e) {
-                        logger.warn("NODE {}: Invalid item type defined {} for {}. Assuming DecimalType.", nodeId,
+                        logger.debug("NODE {}: Invalid item type defined {} for {}. Assuming DecimalType.", nodeId,
```

Comment:
Probably this should be WARN - this is quite a serious error.

Reply:
Changed to warn